dj/api/nodes.py

```diff
@@ -276,7 +290,11 @@ def delete_a_node(name: str, *, session: Session = Depends(get_session)):
     return Response(status_code=HTTPStatus.NO_CONTENT.value)
 
 
-def cube_materialization_config(cube_node: NodeRevision, config: Dict):
+def build_cube_custom_config(
```

Comment:
nit: do we need the `_custom_` infix here?

Reply:
Nope, I removed it